go/webgo/go.mod

```diff
@@ -0,0 +1,5 @@
+module main
+
+go 1.13
```

Comment:
could you remove it ? (I know it's created by `go`)

Reply:
sorry, I'd better be more precise

only this line, not the whole file

Reply:
Oh, I've added the go.mod file back, and remove `go 1.13` from it.